pkgs/by-name/je/jellyfin-ffmpeg/package.nix

```diff
@@ -8,14 +8,23 @@ let
   version = "7.0.2-5";
 in
 
-(ffmpeg_7-full.override {
+(ffmpeg-headless.override {
   inherit version; # Important! This sets the ABI.
   source = fetchFromGitHub {
     owner = "jellyfin";
     repo = "jellyfin-ffmpeg";
     rev = "v${version}";
     hash = "sha256-cqyXQNx65eLEumOoSCucNpAqShMhiPqzsKc/GjKKQOA=";
   };
+
+  # jellyfin-ffmpeg needs OpenCL or CUDA (depending on the GPU) for fully
+  # HW-accelerated video transcoding
+  # https://github.com/NixOS/nixpkgs/issues/271863#issuecomment-1837900239
+  withOpencl = true;
+  withCudaLLVM = true;
```

Comment:
We could actually think about enabling OpenCL in regular ffmpeg too; it appears it's basically header-only.

CUDA via LLVM pulls in LLVM though of course which is quite large and not usually pulled in by many programs.

Reply:
https://github.com/NixOS/nixpkgs/pull/354937